stash/src/scraper/PoolsScraper.ts

```diff
@@ -39,7 +42,7 @@ const getPools = async (block: Block): Promise<PoolEntry[]> => {
       liquidityAssetsInPool[1].toString(),
     ])
     const entry: PoolEntry = {
-      id: liquidityPoolId.unwrap().toNumber(),
+      id: toHuman(liquidityPoolId).toNumber,
```

Comment:
careful! 
As far as I remember, when ids are larger than 999 ( 1000 ), toHuman returns "1,000" and toNumber may lead to incorrect values!
plz check :kissing_heart:

Reply:
Refactored and tried with 1245 value it returns 1245 as it should. Thanks!